ideascube/templates/ideascube/includes/tag_cloud.html

```diff
@@ -1,7 +1,7 @@
 {% load i18n ideascube_tags %}
 {% spaceless %}
 <div class="card tinted">
-    <h4><span class="theme learn">{% trans "browse" %}</span> {% trans "Topics" %}</h4>
+    <h4><span class="theme discover">{% trans "browse" %}</span> {% trans "Topics" %}</h4>
```

Comment:
That was a ask from @swbsf from internal BSF people (we should have open a bug for that)

Before : 
![capture_tag_different_color](https://cloud.githubusercontent.com/assets/86161/22550621/37f2f968-e951-11e6-9692-bed961d5f472.png)

After :
![capture_tag_same_color](https://cloud.githubusercontent.com/assets/86161/22550622/37f3296a-e951-11e6-801f-ff39d21cfec3.png)